src/theme/icons/charms.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export default {
+  path: (
+    <svg
```

Comment:
You don't need an svg tag here. You can move the properties down to the `<path>` tag. Pull the `viewBox` out into it's own top level property like you did on `npcs.js`, and add the `fill="currentColor"` attribute please. :)